include/can/core/messages.hpp

```diff
@@ -215,18 +214,66 @@ struct ReadFromEEPromResponse : BaseMessage<MessageId::read_eeprom_response> {
     bool operator==(const ReadFromEEPromResponse& other) const = default;
 };
 
+using GetMoveStatusRequest = Empty<MessageId::get_move_status_request>;
+
+struct GetMoveStatusResponse
+    : BaseMessage<MessageId::get_move_status_response> {
+    uint8_t group_id;
+    uint8_t seq_id;
+    uint64_t remaining_duration;
+    uint64_t current_position;
+    uint8_t node_id;
+    uint8_t padding;
+
+    template <bit_utils::ByteIterator Input, typename Limit>
+    static auto parse(Input body, Limit limit) -> GetMoveStatusResponse {
+        uint8_t group_id = 0;
+        uint8_t seq_id = 0;
+        uint64_t remaining_duration = 0;
+        uint64_t current_position = 0;
+        uint8_t node_id = 0;
+        uint8_t padding =
```

Comment:
Do you strictly need the padding on the CPP side?

Reply:
I thought it might be easier to debug if the padding is on the CPP side as well, but we can totally omit it.